tools/cldr/genlist.js

```diff
@@ -321,14 +321,6 @@ console.log("\n\nMerging formats forward ...");
 
 aux.mergeFormats(localePatterns, localePatterns, []);
 
-console.log("\n\nPromoting sublocales ...");
```

Comment:
Why is this removed?

Reply:
I realized the data have generated wrongly due to this line. 
It overrides fallback locale data. for example, en-CA data overrides en data which should maintain it.